src/commands/withdraw.js

```diff
@@ -0,0 +1,59 @@
+const { OOPS_TEXT } = require('../messages')
+
+const WITHDRAW_TEXT = 'Wow. Successful withdrawal.'
+const PROPER_AMOUNT_TEXT = 'You need provide a proper amount to be send.'
+const NO_COMMA_TEXT = 'Please avoid "," in your amount and use "."'
+const NEED_ADDRESS_TEXT = 'Need an address as a third argument'
+const NO_FUNDS = 'You dont have doge to transfer.'
+const NOT_ENOUGH_FUNDS = 'Not enough funds for this transfer. Please add some dogecoins.'
```

Comment:
I think we could move theses const to the message file, since some of them are already used somewhere else in the app

Reply:
Start new issue ?